code/game/machinery/alarm.dm

```diff
@@ -166,6 +166,7 @@
 /obj/machinery/alarm/proc/first_run()
 	area_uid = areaMaster.uid
 	name = "[areaMaster.name] Air Alarm"
+	desc = "An alarm used to control the area's atmospherics systems."
```

Comment:
Put this up in the object definition rather than this proc. The name has to be here because it involves a variable, but this is  always the same.